### PR TITLE
RHOAIENG-12880 - Split Distributed Workloads test suites

### PR DESCRIPTION
**Pull request by jiripetrlik**

### CODE REVIEW

File: ods_ci/tests/Tests/0600__distributed_workloads/0602__workloads_training/test-distributed-workloads-metrics-ui.robot [hunk 1]
```diff
@@ -29,7 +29,7 @@ Verify Workload Metrics Home page Contents
     [Documentation]    Verifies "Workload Metrics" page is accessible from
     ...                the navigation menu on the left and page contents
     [Tags]    RHOAIENG-4837
-    ...       Sanity    DistributedWorkloads
+    ...       Sanity    DistributedWorkloads    WorkloadsTraining
```
[the remark this answers is not in the document]
Comment (jiripetrlik) [reply ods_ci/tests/Tests/0600__distributed_workloads/0602__workloads_training/test-distributed-workloads-metrics-ui.robot:32]: :+1: Yes, it is true. Fixed.

[the remark this answers is not in the document]
Comment (ChughShilpa) [reply ods_ci/tests/Tests/0600__distributed_workloads/0602__workloads_training/test-distributed-workloads-metrics-ui.robot]: Thanks Jiri. 
Could you also please rename the folder name to `0602_training'

[the remark this answers is not in the document]
Comment (jiripetrlik) [reply ods_ci/tests/Tests/0600__distributed_workloads/0602__workloads_training/test-distributed-workloads-metrics-ui.robot]: Done.

**Review by sutaakar — APPROVED** [review]

File: ods_ci/tests/Tests/0600__distributed_workloads/0601__workloads_orchestration/test-smoke.robot [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Documentation     Smoke tests for Workloads Orchestration
+Library           Process
+Resource          ../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
+
+
+*** Test Cases ***
+Kueue smoke test
+    [Documentation]    Check that Kueue deployment and its service are up and running
+    [Tags]    Smoke
+    ...       DistributedWorkloads
```
Comment (ChughShilpa) [inline ods_ci/tests/Tests/0600__distributed_workloads/0601__workloads_orchestration/test-smoke.robot:11]: `WorkloadsOrchestration` tag is missed here

**Comment by ChughShilpa** [comment]
Distribution of [upgrade tests](https://github.com/red-hat-data-services/ods-ci/tree/master/ods_ci/tests/Tests/0200__rhoai_upgrade) is missing

**Comment by ChughShilpa** [comment]
How are we going to distribute [UI test suite](https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Tests/0600__distributed_workloads/test-distributed-workloads-metrics-ui.robot) ? 
@sutaakar @jiripetrlik 

As mentioned in [Distributed Workloads - Classification](https://docs.google.com/document/d/146FNqtKZAnjYjKi0wxCyeHLscWsqNC5Or6y7tIP5_kk/edit) , UI/UX work will be classified according to the components classification

**Comment by jiripetrlik** [comment]
@ChughShilpa 
I've put UI tests to Training for now as it is not connected to SDK or Kueue in my opinion.

File: ods_ci/tests/Tests/0600__distributed_workloads/0601__workloads_orchestration/test-smoke.robot (hunk 2 above)
Comment (jiripetrlik) [reply ods_ci/tests/Tests/0600__distributed_workloads/0601__workloads_orchestration/test-smoke.robot:11]: Fixed.

**Comment by sutaakar** [comment]
UI is somewhere in between as it use data from both KubeRay and Kueue.

**Comment by jiripetrlik** [comment]
@sutaakar @ChughShilpa 
I can create a separate folder for UI tests and move it upwards. Can also put there tags for both teams.

**Comment by ChughShilpa** [comment]
> @sutaakar @ChughShilpa I can create a separate folder for UI tests. Can also put there tags for both teams. What do you think?

I think for now we can keep it as it is and can add tags for both components

**Comment by jiripetrlik** [comment]
> > @sutaakar @ChughShilpa I can create a separate folder for UI tests. Can also put there tags for both teams. What do you think?
> 
> I think for now we can keep it as it is and can add tags for both components

Done.

**Review by ChughShilpa — APPROVED** [review]
lgtm

**Review by sutaakar — APPROVED** [review]

**Pull request merged**